app_dart/lib/src/service/scheduler.dart

```diff
@@ -430,25 +434,37 @@ class Scheduler {
   /// Get an aggregate of LUCI presubmit builders from .ci.yaml and try_builders.json.
   Future<List<LuciBuilder>> getPresubmitBuilders({@required Commit commit, int prNumber}) async {
     // Get try_builders.json builders
+    log.debug('Getting presubmit builders from json file');
     final List<LuciBuilder> builders = await config.luciBuilders(
       'try',
       commit.slug,
       commitSha: commit.sha,
     );
     //  Get .ci.yaml targets
+    log.debug('Getting presubmit builders .ci.yaml');
     final SchedulerConfig schedulerConfig = await getSchedulerConfig(commit);
     if (!schedulerConfig.enabledBranches.contains(commit.branch)) {
       throw Exception('${commit.branch} is not enabled for this .ci.yaml.\nAdd it to run tests against this PR.');
     }
+    log.debug('About to get presubmit luci targets');
     final Iterable<Target> presubmitLuciTargets =
         getPreSubmitTargets(commit, schedulerConfig).where((Target target) => target.scheduler == SchedulerSystem.luci);
+    log.debug('About to create luci builder objects');
     final Iterable<LuciBuilder> ciYamlBuilders =
         presubmitLuciTargets.map((Target target) => LuciBuilder.fromTarget(target, commit.slug));
     builders.addAll(ciYamlBuilders);
     // Filter builders based on the PR diff
     final GithubService githubService = await config.createGithubService(commit.slug);
+    log.debug('About to get list of files');
+    githubService.setLogger(log);
     final List<String> files = await githubService.listFiles(commit.slug, prNumber);
-    return await getFilteredBuilders(builders, files);
+    log.debug('About to filter filter builds');
```

Comment:
Nit: duplicated `filter`.

Reply:
Removed the additional logging. This was here only to identify where we were getting the error.

Reply:
Oh, I meant there are two duplicated words `filter` in the log message.